frontend/src/features/properties/list/FilterBar.test.tsx

```diff
@@ -40,7 +40,7 @@ describe('FilterBar', () => {
     );
```

Comment:
Will come back to this file on the weekend, need to adjust for the auto complete component which is a bit more tricky. Have some higher priority bugs I want to do today first.